### PR TITLE
Update game over message

### PR DESCRIPTION
**Pull request by cadwithw**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by cadwithw — COMMENTED** [review]
eeeeeeeeeeeeeeee

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (notxtr) [inline index.html:51]: This is a comment.

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
```
Comment (notxtr) [inline index.html:50]: This is another comment

File: index.html [hunk 3]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
...
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (notxtr) [inline index.html:50]:
```suggestion
<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment (notxtr) [inline index.html:50]:
```suggestion
<h2 hidden>The Game is over, refresh to play again 🧑‍💻 🤖!</h2>
```

**Pull request merged**